data-types/02-intro-vector-data.py

```diff
@@ -0,0 +1,288 @@
+
+# ## Plotting Vector Data with GeoPandas
+#
+# Once your vector data is successfully loaded into a GeoDataFrame with GeoPandas, it's very easy to plot the data with the GeoPandas function `plot()`. This helps to visually demonstrate the data stored in a GeoDataFrame. Below is the data we opened above plotted out. 
+
+# +
+# Using matplotlib to plot the two GeoDataFrames on the same axes (ax)
+fig, ax = plt.subplots()
+
+countries.plot(ax=ax)
+boundary_lines.plot(ax=ax, color='red');
+ax.set(title="Countries plotted with political boundaries overlaid.");
+# -
+
+# <div class="notice--warning" markdown="1">
+#
+# ## <i class="fa fa-pencil-square-o" aria-hidden="true"></i> Challenge : Open and Plot Spatial Data in Python
+#
+# Much like was done above, you're going to open your own spatial data with `geopandas`! Below we've provided a path to a dataset that contains spatial data similar to what we opened above. Open the spatial data with `geopandas`. Name the dataset `populated_places`. Once the dataset is opened, check the `geom_type`, and view the first few rows of the dataset using `dataset.head()`. 
+#
+# </div>
+
+# +
+populated_places_path = os.path.join("data", "earthpy-downloads",
+                                     "ne_50m_populated_places_simple", "ne_50m_populated_places_simple.shp")
+
+# Open your dataset below this line. Make sure to view the geom_type and the first few rows of the dataset
+
+
+# -
+
+# The cell below includes a set of tests to see if you correctly completed the activity in the cell above. They will provide you with feedback that can help you complete the activity. 
+#
+# Be sure to run the cell below to check your code (please do not modify the cell!).
+
+# +
+# Run this cell to ensure soap_plot_locations was created correctly
+
+import notebook_tests_data_types
+
+try:
+    print(notebook_tests_data_types.test_geopandas_dataframe_creation(
+        populated_places)
+    )
+except NameError:
+    print("'populated_places' is not defined. Make sure you spelled the variable name correctly!")
+# -
+
+# Now that the data is open, you can plot is using the `plot()` function. Copy the code below, but enter in your variable name to see the data you've opened plotted. Make sure to customize the title of the code, and change the color of your data to make the map more legible. 
+#
+# ```
+# fig, ax = plt.subplots()
+#
+# countries.boundary.plot(ax=ax)
+# your_dataframe_here.plot(ax=ax, color='blue');
+# ax.set(title="Put Title Here");
+# ```
+
+
+
+# <div class="notice--warning" markdown="1">
```

Comment:
NOTE - i ran out of steam here! Feel free to edit everywhere but i know this section needs a bit more work.